translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [:el: Greek](README.greek.md)  |
```

Comment:
The flag doesn't show: 

![image](https://user-images.githubusercontent.com/5722022/66915623-c0bd7080-f019-11e9-9263-6b3073497c0d.png)

If you don't find a flag that works, you can remove it.

Reply:
I added a working flag.